docs/source/notebooks/DiCE_with_advanced_options.ipynb

```diff
@@ -16,9 +16,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "D:\\xiaoqi\\anaconda3\\envs\\pytorch-py38\\lib\\site-packages\\dice_ml\\utils\\exception.py:12: UserWarning: UserConfigValidationException will be deprecated from dice_ml.utils. Please import UserConfigValidationException from raiutils.exceptions.\n",
```

Comment:
Could you please cleanup the notebook outputs? 

It is hard to know what is the issue that you are trying to fix.

Reply:
Sure, I had removed it.